ansible/README.md

```diff
@@ -14,6 +14,8 @@ sudo pip install ansible==2.0.2.0
 
 #### Vagrant users
 Nothing to be done, Ansible is already installed during vagrant provisioning.
+You can skip setup and prereq steps as those have been done by vagrant for you.  
+You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
 #### Mac users
 It is assumed that a VM has been provisioned using **docker-machine**.
```

Comment:
You're right - can we get rid of this:

`docker-machine ssh $MACHINE_NAME "echo DOCKER_HOST=\'-H tcp://0.0.0.0:4243\' |sudo tee -a /var/lib/boot2docker/profile > /dev/null"`